src/Server.php

```diff
@@ -124,6 +124,7 @@ public function __construct(
         $this->host = $host;
         $this->port = $port;
         $this->ipcSocketPath = $ipcSocketPath;
+        $this->timers = new TimerCollection();
```

Comment:
Why is this line moved? Seems to have nothing to do with the headers...

Reply:
I added this fix to my fork, to make timers work. Ref #40.
So this change will no make any difference.